service/awsconfig/v7/templates/cloudformation/guest/lifecycle_hooks.go

```diff
@@ -0,0 +1,13 @@
+package guest
+
+const LifecycleHooks = `{{define "lifecycle_hooks"}}
+NodeDrainerLifecycleHook:
+  Type: "AWS::AutoScaling::LifecycleHook"
+  Properties:
+    AutoScalingGroupName:
+      Ref: {{ .LifecycleHooks.NodeDrainer.Name }}
+    DefaultResult: Continue
+    HeartbeatTimeout: 300
+    LifecycleHookName: NodeDrainer
+    LifecycleTransition: "autoscaling:EC2_INSTANCE_TERMINATING"
+{{end}}`
```

Comment:
So this seems to be wrong. The docs are a bit conflicting since they state different templates at the top and at the bottom of the document. See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-as-lifecyclehook.html. Right now I get this error. 

```
[
  {/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/framework/framework.go:493: } 
  {/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/framework/resource/metricsresource/resource.go:153: } 
  {/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/framework/resource/retryresource/resource.go:188: } 
  {/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/framework/resource/retryresource/resource.go:176: } 
  {/go/src/github.com/giantswarm/aws-operator/service/awsconfig/v7/resource/cloudformation/create.go:29: } 
  {ValidationError: Invalid template property or properties [NodeDrainerLifecycleHook]\n\tstatus code: 400, request id: 8360fae7-1d45-11e8-b480-efffe51b34f9}
]
```

@rossf7 @fgimenez any idea? My research so far was not very successful on finding out how the templates should actually look like.

Reply:
I think a problem is the indentation of the sub template. I fixed that and try again.